src/coral_models/prepare_raw_data.py

```diff
@@ -0,0 +1,470 @@
+        lambda x: x.replace("subject_a_", ""), axis=1
+    )
+    speaker_b = speaker_metadata[speaker_b_columns].rename(
+        lambda x: x.replace("subject_b_", ""), axis=1
+    )
+    recorder = speaker_metadata[recorder_columns].rename(
+        lambda x: x.replace("recorder_", ""), axis=1
+    )
+
+    speaker_list = [speaker_a, speaker_b, recorder]
+
+    # Load speaker information from read aloud data
+    read_aloud_paths = raw_path.glob("*_oplæst_*")
+    for read_aloud_path in read_aloud_paths:
+        cnx = sqlite3.connect(read_aloud_path / "db.sqlite3")
+        read_aloud_data = pd.read_sql_query("SELECT * FROM CoRal_recording", cnx)
+        read_aloud_data_speakers = read_aloud_data[db_to_excel_metadata_names.keys()]
+        read_aloud_data_speakers = read_aloud_data_speakers.rename(
+            columns=db_to_excel_metadata_names
+        )
+        speaker_list.append(read_aloud_data_speakers)
+
+    # Concatenate all speaker information
+    speakers = pd.concat(speaker_list, axis=0).drop_duplicates().reset_index(drop=True)
+
+    # People often typed their name in all lower case or similiar variations, which
+    # yields a lot of duplicates. We therefore we remove rows with dublicate emails.
+    speakers = speakers.drop_duplicates(subset=["mail"])
+
+    # The native_language columns contains both aplha-2 codes and full names, so we
+    # need to convert the aplha-2 codes to full names. The same goes for the
+    # birthplace. Spoken_languages is a list of languages, so we need to convert that
+    # as well.
+    speakers["native_language"] = speakers["native_language"].apply(correct_language)
+    speakers["birthplace"] = speakers["birthplace"].apply(correct_language)
+    speakers["spoken_languages"] = speakers["spoken_languages"].apply(
+        correct_country_list
+    )
+
+    # Specification of gender is not consistent, so we need to correct that
+    # by mapping "K" to "female", and "M" to "male".
+    speakers["gender"] = speakers["gender"].apply(
+        lambda x: "male"
+        if x in ["male", "M"]
+        else "female"
+        if x in ["K", "female"]
+        else x
+    )
+
+    # Create a speaker id column
+    speakers["speaker_id"] = "t" + speakers.index.astype(str)
+
+    return speakers
+
+
+def make_recording_metadata(
+    cfg: DictConfig, speakers: pd.DataFrame, raw_path: Path, sentences: pd.DataFrame
+) -> pd.DataFrame:
+    """Make a recording metadata file from the raw data
+
+    Args:
+        cfg (DictConfig):
+            The config
+        speakers (pd.DataFrame):
+            The speakers dataframe
+        raw_path (Path):
+            The path to the raw data
+        sentences (pd.DataFrame):
+            The sentences dataframe
+
+    Returns:
+        pd.DataFrame: The recording metadata dataframe"""
+    # Load the metadata
+    metadata_path = raw_path / "metadata.xlsx"
+
+    # Make speaker-metadata file
+    speaker_metadata = pd.read_excel(metadata_path)
```

Comment:
You don't want to remove the "Unnamed" columns this time?

```suggestion
    speaker_metadata = pd.read_excel(metadata_path)
```